ipalib/util.py

```diff
@@ -170,6 +170,10 @@ def check_writable_file(filename):
     """
     if filename is None:
         raise errors.FileError(reason=_('Filename is empty'))
+
+    if os.path.isdir(filename):
+        raise errors.FileError(reason=_('Directory name found instead of '
+                                        'Filename'))
```

Comment:
The error does not say much and has `F` in "filename" uppercased for some reason, could you rather go with something like "Expected file but {fname} is a directory".format(fname=filename)?